renderers/chords_renderer.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+const parseVerse = require('../verse.js')['parseVerse'];
+
+class ChordsRenderer {
+  constructor(chordChartJson) {
+    this.chordChartJson = chordChartJson;
+  }
+
+  createHtmlChordChart() {
+    const chordChartHtml = document.createElement('div');
+    chordChartHtml.className = 'chart';
+
+    this.chordChartJson.forEach((verse) => {
```

Comment:
Each element in the verse (chordChartJson) is called a phrase. Rename verse -> phrase.